### DISCUSSION TITLE
Can we have NetStandard compiled version of CefSharp.OffScreen ?

### DISCUSSION THREAD

**Discussion by TejasPatadiya10** [cefsharp/CefSharp#4774] [General]
It would be good if we have CefSharp.OffScreen compiled in netstandard2.0 and greater as we can use the same in framework and core.

**Comment by amaitland** [comment]
> C++/CLI projects can't target .NET Standard.
C++/CLI projects don't support the newer SDK-style project file format. Instead, C++/CLI projects use the same .vcxproj file format that other Visual Studio C++ projects use.
C++/CLI projects can't target multiple .NET platforms. If you need to build a C++/CLI project for both .NET and .NET Framework, use separate project files.

https://learn.microsoft.com/en-us/dotnet/core/porting/cpp-cli#ccli-net-core-limitations

Unfortunately we cannot target .net standard.

> as we can use the same in framework and core.

There's nothing stopping you now. You just need to multi target your library.

See https://github.com/cefsharp/CefSharp.Wpf.HwndHost/blob/master/CefSharp.Wpf.HwndHost/CefSharp.Wpf.HwndHost.csproj#L25 for an example.
